services/brig/test/integration/API/User/Util.hs

```diff
@@ -575,3 +576,12 @@ nonce m brig uid cid =
         . paths ["clients", toByteString' cid, "nonce"]
         . zUser uid
     )
+
+createAccessToken :: (MonadIO m, MonadHttp m, HasCallStack) => Brig -> UserId -> ClientId -> Proof -> m ResponseLBS
+createAccessToken brig uid cid proof =
+  post
+    ( brig
+        . paths ["clients", toByteString' cid, "access-token"]
```

Comment:
Tangential:

Considering we have lots of RPC calls between services, would it make sense to start documenting these kind of calls to make it easier to jump from here to wherever this call is handled inside Brig? Probably not as important inside test files.